gwctl/pkg/printer/policies.go

```diff
@@ -148,3 +150,65 @@ func (pp *PoliciesPrinter) PrintDescribeView(policies []policymanager.Policy) {
 		}
 	}
 }
+
+type policyCrdDescribeView struct {
+	// PolicyCrd   *apiextensionsv1.CustomResourceDefinition `json:",omitempty"`
+	Name        string                 `json:",omitempty"`
+	Namespace   string                 `json:",omitempty"`
+	Labels      map[string]string      `json:",omitempty"`
+	Annotations map[string]string      `json:",omitempty"`
+	APIVersion  string                 `json:",omitempty"`
+	Kind        string                 `json:",omitempty"`
+	Metadata    map[string]interface{} `json:",omitempty"`
+	Spec        map[string]interface{} `json:",omitempty"`
```

Comment:
Will using the concrete type here instead of `map[string]interface{}` make things simple? Any reason why we purposely want to keep this generic?

Same comment for Metadata and Status

Reply:
Yes, initially I used concrete types itself, but the output was not being parsed properly by `yaml` package, so I had to convert it to `map[string]interface{}`.
Should I leave it as it is or do you suggest any other alternative?

Reply:
Can you elaborate on what problem you run into if you use something like this:

```
	Metadata    *metav1.ObjectMeta                              `json:",omitempty"`
	Spec        *apiextensionsv1.CustomResourceDefinitionSpec   `json:",omitempty"`
	Status      *apiextensionsv1.CustomResourceDefinitionStatus `json:",omitempty"`
```

I think this should make things _very_ easy. Am I missing something?

Reply:
My mistake, I was not dereferencing the type, due to which it wasnt parsing properly. I works fine now